etc/disable-common.inc

```diff
@@ -172,3 +172,29 @@ blacklist ${PATH}/roxterm-config
 blacklist ${PATH}/terminix
 blacklist ${PATH}/urxvtc
 blacklist ${PATH}/urxvtcd
+
+# disable common suid programms
+blacklist ${PATH}/firejail
```

Comment:
So far it has not broken a single configuration in any of my tests. Can you provide an example of a possible conflict?
This feature is useful in case nonewprivs or no root can not be used or as an additional failsafe.